### ISSUE TITLE
What is the typical loss function value at different stages of training (e.g., early training, mid-training, late training)?

### ISSUE THREAD

**Issue by LazyChild** [TencentARC/T2I-Adapter#125]
Excellent work! I would like to consult about the approximate value of loss during the whole training process. I am using about 20k 512*512 images to train T2I-Adapter-sketch, and the epoch average loss always stays around 0.14. I cannot tell whether it has converged. I want to use the loss in your experiments (especially the initial loss and the loss near convergence) to judge how my experiment is doing.
